### PR TITLE
README: improve documentation on authentication configuration 

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

- improve documentation on authentication configuration 
- reorder to have required configuration information before optional information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,19 @@ The tests depend on having Firefox (default) or Chrome downloaded.
 1. `bundle install`
 1. `bundle exec rake webdrivers:geckodriver:update`
 
-See the `Configuration` section below for instructions on using Chrome and on tweaking default browser settings.
+See the `Authentication Configuration` section below for
+- authentication token instructions (dor_services_app, dor_services, etd)
```
Comment (jcoyne) [inline README.md:15]: I'm not sure what the parenthetical is providing here.

File: README.md [hunk 2]
```diff
@@ -53,46 +58,50 @@ Host *.stanford.edu
     GSSAPIDelegateCredentials yes
 ```
 
-### Problems with Authentication?
+### Set Dor-Services-App and Dor-Services Tokens
```
Comment (jcoyne) [inline README.md:61]: There is only one dor-services-app. dor-services is a gem used by dor-services-app.

File: README.md [hunk 3]
```diff
@@ -53,46 +58,50 @@ Host *.stanford.edu
...
 
-If specs fail because they get through authentication without finding the "duo_iframe" "Send Me a Push", add `automatic_authentication: true` to `config/settings.local.yml`.
+Some integration tests use the `dor-services-client` to interact with the `dor-services-app`, others directly (or indirectly) use the `dor-services-app`.  In order to successfully use the dor-services-app API, you must first have a token set. (To generate dor-services-app tokens, see the [dor-services-app README](https://github.com/sul-dlss/dor-services-app#authentication).)
 
-You may also want to lower the time value of `post_authentication_text_timeout` in `config/settings.local.yml`.
+Place the value in `config/settings/{ENV}.local.yml`. See `config/settings.yml` for the expected YAML syntax. Note that you'll need to do this for each environment (currently: staging and qa), and you'll need top level keys of *both* `dor-services-app` and `dor-services` (same token). See `settings.yml` for explanation.
```
Comment (jcoyne) [inline README.md:65]: The `dor_services_app` key is never used and should probably be removed.

File: README.md (hunk 1 above)
Comment (ndushay) [reply README.md:15]: it indicates that there are 3 diff authentication tokens listed.   I missed "dor_services".

File: README.md (hunk 2 above)
Comment (ndushay) [reply README.md:61]: This isn't about the names of apps, it's about the labels in the yaml file.

File: README.md (hunk 3 above)
Comment (ndushay) [reply README.md:65]: ooo!   I'll try it.

**Comment by ndushay** [comment]
@jcoyne updated again - simpler instructions!

**Review by jcoyne — APPROVED** [review]

**Pull request merged**